app/lib/api/file_system/file_system_dav.dart

```diff
@@ -85,10 +85,18 @@ class DavRemoteDocumentFileSystem extends DocumentRemoteSystem {
       return;
     }
     final xml = XmlDocument.parse(content);
-    final currentElement = xml.findAllElements('d:response').where((element) {
+    var responses = xml.findAllElements('d:response').where((element) {
```

Comment:
please use final if you assign it only once
```suggestion
    final responses = xml.findAllElements('d:response').where((element) {
```